api/src/repositories/survey-critter-repository.ts

```diff
@@ -66,16 +73,15 @@ export class SurveyCritterRepository extends BaseRepository {
    * @returns {*}
    * @memberof SurveyCritterRepository
    */
-  async upsertDeployment(critterId: number, deplyomentId: string): Promise<number> {
+  async upsertDeployment(critterId: number, deplyomentId: string): Promise<void> {
     defaultLog.debug({ label: 'addDeployment', deplyomentId });
+    //This update operation intentionally changes nothing. Only really being done to trigger update audit columns.
```

Comment:
Any important comments like this that convey business logic should be moved to the JS docs.